macro/run_simScript.py

```diff
@@ -394,14 +397,15 @@
  # mu_start, mu_end =  ship_geo.tauMudet.zMudetC,ship_geo.TrackStation2.z
  #
  # in front of UVT up to tracking station 1
- mu_start, mu_end = ship_geo.Chamber1.z-ship_geo.chambers.Tub1length-10.*u.cm,ship_geo.TrackStation1.z
- print('MuDIS position info input=',mu_start, mu_end)
- DISgen.SetPositions(mu_start, mu_end)
+ 
+ DISgen.SetPositions(options.mudis_z_range[0], options.mudis_z_range[1], options.mudis_x_range[0], options.mudis_x_range[1], options.mudis_y_range[0], options.mudis_y_range[1])
  DISgen.Init(inputFile,options.firstEvent)
  primGen.AddGenerator(DISgen)
  options.nEvents = min(options.nEvents,DISgen.GetNevents())
  inactivateMuonProcesses = True # avoid unwanted hadronic events of "incoming" muon flying backward
  print('Generate ',options.nEvents,' with DIS input', ' first event',options.firstEvent)
+ print("MuDIS position nfo input:\nx_range =", options.mudis_x_range, "y_range =", options.mudis_y_range, "z_range = ", options.mudis_z_range)
```

Comment:
Typo `nfo`, space missing after equals signs for `x_range` and `z_range`. Units would be useful.
Line very long.